### ISSUE TITLE
🐛 `experimentalScannerIgnores` doesn't ignore check of biome.json schema version

### ISSUE THREAD

**Issue by Swimburger** [biomejs/biome#7805]
### Environment information

```block
CLI:
  Version:                      2.2.6
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v24.3.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/11.4.2

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Workspace:
  Open Documents:               0
```

### What happened?

When I run `biome check --fix --unsafe`, I still get this warning even tho the biome.json file is ignored in the root biome.json file.
```
The configuration schema version does not match the CLI version
```

Root biome.json:
```json
{
  "$schema": "https://biomejs.dev/schemas/2.2.6/schema.json",
  "root": true,
  "files": {
    "includes": ["**", "!!lib", "!!dist", "!!seed/", "!!generator/asIs/biome.json"],
    "experimentalScannerIgnores": ["seed", "asIs"]
  },
```

It successfully ignores folders with name `seed` and `asIs` in general, but not for the biome.json schema version warning.

Aside: the `!!` syntax doesn't work :(

### Expected result

It should ignore the generator/asIs/biome.json completely

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
I'm going to close the issue, if you don't mind.

The double exclamation syntax will be part of v2.3, so I don't know where you read about it, and why it should work for v2.2.6

**Comment by Swimburger** [comment]
I'm using `experimentalScannerIgnores` for that for now instead of the force ignore syntax. Either way, this is low priority since it's just a warning.
I'll revisit if it doesn't work with force ignore syntax in 2.3

**Comment by Swimburger** [comment]
2.3 works great! 🙇
